ompi/mpi/fortran/use-mpi-f08/base/ts.c

```diff
@@ -0,0 +1,141 @@
+/* -*- Mode: C; c-basic-offset:4 ; -*- */
+/*
+ * Copyright (c) 2014      Argonne National Laboratory.
+ * Copyright (c) 2019      Research Organization for Information Science
+ *                         and Technology (RIST).  All rights reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#include "ts.h"
+
+#include <assert.h>
+
+#if OMPI_FORTRAN_HAVE_TS
```

Comment:
#if'ing out the contents of the entire file will cause weirdness (perhaps a warning) if this file is compiled but empty.  Can we just not compile this file if !OMPI_FORTRAN_HAVE_TS?

Reply:
i'll check on this.

Reply:
moved the conditional into the makefile